homeworks/src/main/java/hw31/jdbc/DBConnection.java

```diff
@@ -0,0 +1,65 @@
+package hw31.jdbc;
+
+import hw31.exception.JdbcOperationException;
+import org.postgresql.ds.PGSimpleDataSource;
+import org.springframework.stereotype.Service;
+
+import javax.sql.DataSource;
+import java.io.IOException;
+import java.io.InputStream;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.Objects;
+import java.util.Properties;
+
+@Service
```

Comment:
All methods in this class are static, you do  not need to add `DBConnection` to the Spring context, remove `@Service` annotation